.evergreen/.evg.yml

```diff
@@ -291,6 +291,7 @@ functions:
           export AWS_TEMP_SESSION_TOKEN=$CSFLE_AWS_TEMP_SESSION_TOKEN
           export CRYPT_SHARED_LIB_PATH=${CRYPT_SHARED_LIB_PATH}
 
+          ASYNC_TRANSPORT="${ASYNC_TRANSPORT}" NETTY_SSL_PROVIDER="${NETTY_SSL_PROVIDER}" \
```

Comment:
Add support for the Netty variants in the main "run tests" function and remove the "run netty tests" function below.